Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift

```diff
@@ -75,9 +84,9 @@ extension SentryFeedback {
     /**
      * - note: Currently there is only a single attachment possible, for the screenshot, of which there can be only one.
      */
-    func attachments() -> [Attachment] {
+    func attachmentsForEnvelope() -> [Attachment] {
```

Comment:
after renaming screenshots to attachments, i had to disambiguate this function name. it's more descriptive anyways so i like it better